BUILDING.md

```diff
@@ -185,7 +185,7 @@ If you are running CMake v3.13 or newer you can build gRPC's dependencies
 in "module" mode and install them alongside gRPC in a single step.
 [Example](test/distrib/cpp/run_distrib_test_cmake_module_install.sh)
 
-If you are using an older version of gRPC, you will need to select "package"
```

Comment:
"older version of gRPC" is way too vague. we need to specify which exact version this applies to. AFAIK the PR enabling "module" for installation hasn't been released yet, so this will only work for 1.27.x and up (1.27.x hasn't been released yet).